### PR TITLE
Fix example controller name to be more coherent and usage of translator

### PR DESCRIPTION
**Pull request by matks**
Fixed example controller name to fit with naming convention (see https://devdocs.prestashop.com/1.7/modules/concepts/controllers/front-controllers/)

Thanks to @BlackKerio for the hint

### CODE REVIEW

File: src/content/1.7/modules/creation/module-translation/new-system.md [hunk 1]
```diff
@@ -140,11 +140,11 @@ Since the module is called MyModule, the translation domain should be `Modules.M
 ```php
 // file: controllers/front/something.php
 
-class MyModuleSomethingFrontController extends ModuleFrontController
+class MySomethingModuleFrontController extends ModuleFrontController
 {
     public function initContent()
     {
-        $this->title = $this->module->trans('My module title', [], 'Modules.Mymodule.Something);
+        $this->title = $this->trans('My module title', [], 'Modules.Mymodule.Something);
```
Comment (matks) [inline src/content/1.7/modules/creation/module-translation/new-system.md]: According to my test, this is the right call. Can someone confirm, maybe @Quetzacoalt91 ?

Comment (PierreRambaud) [reply src/content/1.7/modules/creation/module-translation/new-system.md]: I confirm you this is the right call (according to my test too 😅 )

Comment (Quetzacoalt91) [reply src/content/1.7/modules/creation/module-translation/new-system.md]: When using trans() with a domain, both are valid.  :)

File: src/content/1.7/modules/creation/module-translation/new-system.md [hunk 2]
```diff
@@ -140,11 +140,11 @@ Since the module is called MyModule, the translation domain should be `Modules.M
 ```php
 // file: controllers/front/something.php
 
-class MyModuleSomethingFrontController extends ModuleFrontController
+class MySomethingModuleFrontController extends ModuleFrontController
```
Comment (eternoendless) [inline src/content/1.7/modules/creation/module-translation/new-system.md]: This is wrong. If the module is called `mymodule` then the controller `Something` must be called like this:

```suggestion
class MymoduleSomethingModuleFrontController extends ModuleFrontController
```

> the class name follows this format: `<ModuleName><FileName>ModuleFrontController`.

[source](https://devdocs.prestashop.com/1.7/modules/concepts/controllers/front-controllers/)

File: src/content/1.7/modules/creation/module-translation/new-system.md (hunk 1 above)
Comment (matks) [reply src/content/1.7/modules/creation/module-translation/new-system.md]: @Quetzacoalt91 If I try to use `$this->module->trans()` in a module front controller with PS 1.7.7, I get the following error:
Capture d’écran 2019-12-30 à 12 06 12

File: src/content/1.7/modules/creation/module-translation/new-system.md (hunk 2 above)
Comment (matks) [reply src/content/1.7/modules/creation/module-translation/new-system.md]: Indeed, bad copy-paste

**Comment by matks** [comment]
Thank you for review

**Pull request merged**